Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReport.c

```diff
@@ -1063,10 +1070,11 @@ void bsg_kscrw_i_writeAllThreads(const BSG_KSCrashReportWriter *const writer,
     // Fetch info for all threads.
     writer->beginArray(writer, key);
     {
-        for (mach_msg_type_number_t i = 0; i < numThreads; i++) {
-            thread_t thread = threads[i];
```

Comment:
Now we're no longer using `threads` can we remove lines 1060 - 1070 and 1082 - 1088?

Reply:
I've updated the rest of the codebase to use the new getAllThreads and freeThreads functions. There were actually a couple of places where memory could leak before :)